islandora_paged_content_pdf_batch.drush.inc

```diff
@@ -69,7 +65,7 @@ function islandora_paged_content_pdf_batch_drush_command() {
       ),
       'namespace' => array(
         'description' => dt('The namespace for objects created by this command.  Defaults to namespace set in Fedora config.'),
-        'required' => FALSE,
+        'required' => TRUE,
```

Comment:
Extra helpful text is ++ for sure! If you add another commit reverting the `required` bit I'll get this merged.